modules/ROOT/pages/_partials/api-gov-console-scap-callouts.adoc

```diff
@@ -3,9 +3,9 @@
 image::api-gov-console.png[Screenshot of the API Governance console]
 
 [calloutlist]
-. View a numeric and visual summary of your <<gov-profiles,governance profiles>>, <<governed-apis,governance status>>, <<api-conformance,conformance status>>, and <<nonconformance-severity,nonconformance by severity>>.
-. View, filter, and search a summary list of your governance profiles or governed APIs.
+. View insights to help you monitor your organization's API conformance.
```

Comment:
do we want to call them insights already intentionally?

Reply:
As we discussed in our meeting, I'll change this wording and they'll remove the verbiage at the top of the console that also has the word insights in it.